MSI/Files.wxs

```diff
@@ -176,4 +179,4 @@
     </DirectoryRef>
   </Fragment>
 
-</Wix>
```

Comment:
Would be nice if you wouldn't change this. I know it's a bit stupid, but some editors just don't add a new line at the end of the file. I wouldn't want to have that being switched forth and back.